addons/mail/models/mail_mail.py

```diff
@@ -44,7 +44,7 @@ def default_get(self, fields):
     mail_message_id = fields.Many2one('mail.message', 'Message', required=True, ondelete='cascade', index=True, auto_join=True)
     mail_message_id_int = fields.Integer(compute='_compute_mail_message_id_int', compute_sudo=True)
     body_html = fields.Text('Text Contents', help="Rich-text/HTML message")
-    body_content = fields.Html('Rich-text Contents', sanitize=True, compute='_compute_body_content')
+    body_content = fields.Html('Rich-text Contents', sanitize=True, compute='_compute_body_content', search="_search_body_content")
```

Comment:
It's not realted to `web_editor` but the commit message feels a bit empty so I'd feel weird r+ing this. I guess there is maybe not much to say ?